mlir/test/Conversion/GPUToLLVMSPV/gpu-to-llvm-spv.mlir

```diff
@@ -0,0 +1,179 @@
+// RUN: mlir-opt -pass-pipeline="builtin.module(gpu.module(convert-gpu-to-llvm-spv{subgroup-size=32}))" -split-input-file -verify-diagnostics %s \
+// RUN: | FileCheck --check-prefixes=CHECK-64,CHECK %s
+// RUN: mlir-opt -pass-pipeline="builtin.module(gpu.module(convert-gpu-to-llvm-spv{index-bitwidth=32 subgroup-size=32}))" -split-input-file -verify-diagnostics %s \
+// RUN: | FileCheck --check-prefixes=CHECK-32,CHECK %s
+
+gpu.module @builtins {
```

Comment:
Pass requires `gpu.module` as other similar passes

Reply:
Is a gpu.module really required to get the gpu operations converted to an LLVM call ? Some consumers may not use a gpu module therefore it would be nice if this was not a limitation of the pass.

Reply:
ok this is resolved